README.md

```diff
@@ -1,68 +1,124 @@
+Where `<index_name>` is an assumed existing index within the given technology, and `<index_type>` is either `full_text`, meaning the data object will be read, processed and then submitted to the index, or `metadata` where the metadata triples associated with qualifying data objects will be indexed.
 
-The `<technology>` in the triple references the indexing technology, currently only suppored by `elasticsearch`.  This string is used to dynamically build the policy invocations when the indexing policy is triggered in order to delegate the operations to the appropriate rule engine plugin.
+The `<technology>` in the triple references the indexing technology, currently only suppored by `elasticsearch`. This string is used to dynamically build the policy invocations when the indexing policy is triggered in order to delegate the operations to the appropriate rule engine plugin.
 
 The attribute is configurable within the `plugin_specific_configuration` for the indexing rule engine plugin.
 
 ### Resource Metadata
 
-An administrator may wish to restrict indexing activities to particular resources, for example when automatically ingesting data.  Should a storage resource be at the edge, that resource may not be appropriate for indexing.  In order to indicate a resource is available for indexing it may be annotated with metadata:
+An administrator may wish to restrict indexing activities to particular resources, for example when automatically ingesting data. Should a storage resource be at the edge, that resource may not be appropriate for indexing. In order to indicate a resource is available for indexing it may be annotated with metadata:
 ```
 imeta add -R <resource name> irods::indexing::index true
 ```
-By default, should no resource be tagged it is assumed that all resources are available for indexing.  Should the tag exist on any resource in the system, it is assumed that all available resources for indexing are tagged.
+By default, should no resource be tagged it is assumed that all resources are available for indexing. Should the tag exist on any resource in the system, it is assumed that all available resources for indexing are tagged.
 
-### Plugin Settings
+### Enabling the Indexing Capability Plugins
 
-There are currently three rule engine plugins to configure for the indexing capability which should be added to the `"rule_engines"` section of `/etc/irods/server_config.json`:
+To enable the Indexing capability, prepend the following plugin configuration to the list of rule engines in `/etc/irods/server_config.json`. Plugin-specific configuration options are explained below.
 
-```
+**IMPORTANT: The comments in the JSON structure are there for explanatory purposes and must not be included in your configuration. Failing to follow this requirement will result in the server failing to service requests.**
```

Comment:
consider...    "are there for" -> "are for"

Reply:
"to service requests." -> "to stand up."   ?

Reply:
Done